### DISCUSSION TITLE
"Uncaught ReferenceError: pgFlatpickr is not defined"

### DISCUSSION THREAD

**Discussion by mezler** [Power-Components/livewire-powergrid#1287] [BUG]
I have done everything according to the docs, and whatever I tried (reinstall for example, checking dependencies etc.) PowerGrid does not work properly because of that error. I am using Laravel 10, Livewire 3, and the table is not rendered properly. 
This is how it looks, and keeps having the error: "Uncaught ReferenceError: pgFlatpickr is not defined".
![Screenshot 2023-12-08 at 14 32 25](https://github.com/Power-Components/livewire-powergrid/assets/12034500/13fa4b9f-c8a3-40d1-b2ad-65c977c8a695)

**Comment by abhaypithadiya** [comment]
Hey! Your app.js should look something like this:

```
import flatpickr from "flatpickr";
window.flatpickr = flatpickr;

import "./../../public/assets/vendor/libs/flatpickr/flatpickr.css";
import './../../vendor/power-components/livewire-powergrid/dist/powergrid';
```

Check this out for more details : https://livewire-powergrid.com/table/column-filters.html#filter-datetimepicker

Hope this helps!

**Reply by mezler** [reply]
Thanks, I give it a try!
